src/pages/ContentfulPage.vue

```diff
@@ -305,7 +305,7 @@ export default {
 				query: contentfulEntries,
 				variables: {
 					contentType: 'page',
-					contentKey: args?.route?.meta?.contentfulPage(args?.route)?.trim(),
+					contentKey: args?.route?.meta?.contentfulPage(args?.route, client)?.trim(),
```

Comment:
This is what was needed to get the server-side render correct.

Reply:
@emuvente Thank you very much!